src/prerender.ts

```diff
@@ -245,6 +248,13 @@ export async function prerender(nitro: Nitro) {
     interval: nitro.options.prerender.interval,
   });
 
+  await nitro.hooks.callHook("prerenderer:done", {
+    nitro,
+    prerenderer: nitroRenderer,
```

Comment:
For consistency, we shall pass `prerenderer` as first arg. Also main `nitro` instance is already accessible we don't need to pass it here.

Reply:
I did think this, but then the `prerender:generate` hook expose a nitro instance as well so wasn't sure the convention.

Reply:
I added the `prerenderedRoutes` if we're going in this direction, I think it's better than accessing it off the nitro instance with the internal flag